Code/GraphMol/RGroupDecomposition/RGroupGa.cpp

```diff
@@ -294,7 +294,7 @@ GaResult RGroupGa::run(int runNumber) {
                  });
   BOOST_LOG(rdInfoLog) << "Run " << runNumber << " Execution "
                        << timeInfo(startTime) << std::endl;
-  GaResult result(best->getFitness(), permutations);
+  GaResult result{best->getFitness(), permutations};
```

Comment:
I find this change confusing as well, I tend to use {} for initializer lists which this is clearly not.  But that is my style, however I think we should be consistent.